files/install-requirements-pip.yaml

```diff
@@ -9,7 +9,7 @@
         name: "{{ item }}"
       with_items:
         - ogr
-        - rebasehelper
```

Comment:
How far are we from removing rebase-helper as a dependency of packit (RPM)?

Reply:
It should be already done.

Reply:
https://github.com/packit/packit/commit/988088cd9efaf8906933315d9f42aa6f26996e62#diff-b1ede3399cc051f4d7b1ad8726131e155f6904d5c40bb5e7d4a867cc8de3a54a

Reply:
But I found some remains and references to rebase-helper in comments etc., I'll clean it up in a different PR.